openhtf/plugs/usb/__init__.py

```diff
@@ -36,56 +38,153 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('device_type', 'android, fastboot or general usb type')
+conf.Declare('usb_hub', 'local or Ethersyncy')
```

Comment:
usb_hub is a poor name, i think you're trying to group usb devices into a single top-level conf key.  i don't think this even actually belongs in config, though, because config is intended to capture the types of things that vary between multiple instances of the same tester.  in this case, usb class/subclass/protocol/vid/pid are really baked into the device, so should be hardcoded instead.